### PR TITLE
Handle primitive types

### PR DESCRIPTION
**Pull request by ChloePlanet**
## Changes
|before|after|
|---|---|
|image|image|

### CODE REVIEW

**Review by cuminandpaprika — APPROVED** [review]
LGTM, great work finding a better way to represent it with PUML btw

File: pkg/catalogdiagrams/diagrams.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func (v *DataModelView) GenerateDataView(dataParam *DataModelParam, appName stri
 			}
 			v.DrawPrimitive(viewParam, pe.String(), relationshipMap)
 		} else if seq := entityType.GetSequence(); seq != nil {
-
+			// TODO
```
Comment (orlade-anz) [inline pkg/catalogdiagrams/diagrams.go]: Nit: elaborate or remove

**Review by orlade-anz — CHANGES_REQUESTED** [review]
Without having inspected the logic, this seems wrong for the case given in the PR. If a `FooRequest` does have a single `id` field which is a `string`, will this behaviour be applied?

File: pkg/catalogdiagrams/diagrams.go (hunk 1 above)
Comment (ChloePlanet) [reply pkg/catalogdiagrams/diagrams.go]: Forget to push the updated demo file.
The original sysl spec is 
```
    !alias FooRequest:
        string
```
so it's supposed to have no fields. But the old implementation is add an `id` as placeholder.

**Pull request merged**